build/build.sh

```diff
@@ -46,4 +46,4 @@ if [[ "${GITHUB_EVENT_NAME}" == "push" ]]; then
 else
     echo "${GITHUB_EVENT_NAME} build"
     ${SBT_CMD}
-fi
+fi
```

Comment:
revert change

Reply:
Yea I had some stuff locally from testing this. Lemme fix real quick.